src/clj/game/cards/resources.clj

```diff
@@ -139,6 +139,19 @@
                                  (shuffle! :deck)
                                  (runner-install target))}]}
 
+   "Assimilator"
```

Comment:
Might be worth adding code to handle turning face-down Console and Events. This might crash / cause major issues on turning over (`card-init`) Events..?

Reply:
Right, forgot about those. Knew I could count on you 😄 .